test.js

```diff
@@ -168,4 +169,41 @@ describe('breakdancer', function () {
       assume(viewport.height).equals(breakdancer.height());
     });
   });
+
+  describe('#compare', function () {
+    var bd = new Breakdancer(specification, {
+      innerWidth: 1234,
+      innerHeight: 1000,
+      document: {
+        documentElement: {
+          clientHeight: 1337,
+          clientWidth: 1338
+        }
+      }
+    });
+
+    it('should throw an error when looking at an unspecified breakpoint', function () {
+      assume(bd.compare('hologram', 'width')).throws(TypeError);
+    });
+
+    it('should return the current height when no breakpoint property is specified', function () {
+      assume(bd.compare('whatever', 'height')).throws(TypeError);
+    });
+
+    it('should return the difference in width between the current and specified breakpoint', function () {
+      assume(bd.compare('mobile', 'width')).equals(1234 - 400);
+      assume(bd.compare('mobile', 'height')).equals(1000 - 600);
+    });
+  });
+
+  describe('#compare{Height,Width}', function () {
+    var bd = new Breakdancer(specification);
+    var spy = sinon.spy(bd, 'compare');
+
+    it('should call compare', function () {
+      bd.compareHeight('mobile');
+      bd.compareWidth('mobile');
+      assume(spy.callCount).equals(2);
```

Comment:
might want to assert the functions are called with the right property e.g. `width` and `height`